react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+function customButton() {
+  return <button>Click Me!</button>;
+}
+
+ReactDom.render(customButton(), document.querySelector('#root'));
```

Comment:
With components we don't want to directly call them like a normal function. We use them as a JSX element:

```JSX
ReactDom.render(
  <CustomButton />,
  document.querySelector('#root')
);
```